src/Tools/ExternalAccess/OmniSharp/Internal/PickMembers/OmniSharpPickMembersService.cs

```diff
@@ -26,7 +26,7 @@ public OmniSharpPickMembersService(IOmniSharpPickMembersService omniSharpPickMem
 
         public PickMembersResult PickMembers(string title, ImmutableArray<ISymbol> members, ImmutableArray<PickMembersOption> options = default)
         {
-            var result = _omniSharpPickMembersService.PickMembers(title, members, options.SelectAsArray(o => new OmniSharpPickMembersOption(o)), selectAll: true);
+            var result = _omniSharpPickMembersService.PickMembers(title, members, options.IsDefault ? default : options.SelectAsArray(o => new OmniSharpPickMembersOption(o)), selectAll: true);
```

Comment:
up do yuo.  however, another idiom is: 
```suggestion
            var result = _omniSharpPickMembersService.PickMembers(title, members, options.NullToEmpty().SelectAsArray(o => new OmniSharpPickMembersOption(o)), selectAll: true);
```

They will be effectively teh same cost, and i personally think the latter is nicer to read.

Reply:
These are different semantics. I'd prefer to pass exactly what was originally given, as default is not empty (which is the cause of the bug in the first place.

Reply:
This will pass `ImmutableArray<T>.Empty` instead of `default(ImmutableArray<T>)`, which may or may not have the same behavior in the implementation of `PickMembers`.

Reply:
so as the author of the PickMembers API (and VS) impl i can firmly state that allowing 'default' was never part of the intent of it.  How is 'default' even flowing in here?  It would crash VS if that was happening, so we have a deeper issue if we need to solve that :)

Reply:
ignore tha tprevious line.  this is about 'options', not 'members'.  The original intent is that 'default' is exactly equivalent to empty.  We even enforce that in the VS layer with:

```c#
            options = options.NullToEmpty();
```

:)

It's just optional so it doesn't have to be passed along at the callsite.